boot-qemu.sh

```diff
@@ -137,8 +155,11 @@ function setup_qemu_args() {
         arm32_v7)
             ARCH=arm
             APPEND_STRING+="console=ttyAMA0 "
+            # https://lists.nongnu.org/archive/html/qemu-discuss/2018-08/msg00030.html
+            # VFS: Cannot open root device "vda" or unknown-block(0,0): error -6
+            ${DEBIAN} && HIGHMEM=,highmem=off
```

Comment:
should the value assigned to `HIGHMEM` be a quoted string?

Reply:
`shellcheck` does not complain but I can update that if you feel it is worthwhile.